code/game/objects/structures/crates_lockers/closets/bodybag.dm

```diff
@@ -46,6 +46,12 @@
 		return 1
 	return 0
 
+/obj/structure/closet/body_bag/handle_lock_addition()
```

Comment:
All of these functions you wrote literally do the default behavior. What you want to do here is just a `return`, not calling the parent function.

Reply:
The `..()` is here so that you hit the bodybag / cardboard / whatever, instead of it `return`ing and doing nothing. I can change that behaviour if you'd like?

Reply:
The `..()` is literally redundant if it's here on its own. That is the default behavior of an inherited proc.

Do what you want, just remove the stuff that just states the obvious. :P 

Do hit me up on Discord if you need a deeper explanation, I'm happy to help out.

Reply:
Heh no worries, I think my brain wasn't quite engaged when I did this :P